app-shells/direnv/direnv-2.14.0.recipe

```diff
@@ -0,0 +1,67 @@
+SUMMARY="Unclutter your .profile"
+DESCRIPTION="direnv is an environment switcher for the shell. It knows \
+how to hook into bash, zsh, tcsh and fish shell to load or unload \
+environment variables depending on the current directory. This allows \
+project-specific environment variables without cluttering the ~/.profile file."
+HOMEPAGE="https://direnv.net/"
+COPYRIGHT="2017 zimbatm and contributors"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/direnv/direnv/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="917838827cb753153b91cb2d10c0d7c20cbaa85aa2dde520ee23653a74268ccd"
+PATCHES="direnv-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+
+PROVIDES="
+	direnv = $portVersion
+	cmd:direnv = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+
+BUILD_PREREQUIRES="
+	cmd:gcc
+	cmd:git
+	cmd:go
+	cmd:go_md2man
+	cmd:make
+	cmd:makeinfo
+	cmd:which
+	"
+
+BUILD()
+{
+	# setup go environment
+	BUILD=$(pwd)/build
+	BUILD_GOPATH=$BUILD/gopath
+	BUILD_GOROOT=$BUILD/goroot
+
+	rm -rf $BUILD
+	mkdir -p $BUILD_GOPATH/bin $BUILD_GOPATH/src $BUILD_GOROOT
+	cp -R -s $(go env GOROOT)/* $BUILD_GOROOT
+
+	export GOPATH=$BUILD_GOPATH
+	export GOROOT=$BUILD_GOROOT
+	export PATH=$PATH:$GOPATH/bin
+
+	# copy additional packages
+	cp -R -s $(pwd)/vendor/* $GOPATH/src
+
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	install -d $binDir
+	install -d $manDir
+	install /usr/local/bin/* $binDir
+	mv /usr/local/share/man/* $manDir
+}
```

Comment:
Here's a better way to handle `INSTALL()` based on the [`install`][0] target

```diff
 INSTALL()
 {
-	make install
-
-	install -d $binDir
-	install -d $manDir
-	install /usr/local/bin/* $binDir
-	mv /usr/local/share/man/* $manDir
+	install -d $binDir $manDir/man1
+	install -t $binDir direnv
+	install -t $manDir/man1 man/*.1
 }
```

[0]: https://github.com/direnv/direnv/blob/99e43f697ca10be5ec28b0185f9cfd9c456dc527/Makefile#L54